script/build_hosts

```diff
@@ -14,16 +14,16 @@ if args.wf:
     name = "wf2023"
     detail = "2023Sharm"
     timezone = "Africa/Cairo"
-    max_team = 152
-    max_human_judge = 12
-    max_pc2_autojudge = 16
+    max_team = 140
+    max_human_judge = 14
+    max_pc2_autojudge = 20
     max_kattis_autojudge = 0
-    max_domjudge_autojudge = 16
-    max_domjudge_ccsadmin = 7
-    # pc2 for wf2021
-    max_ccsadmin = 4
+    max_domjudge_autojudge = 24
+    max_domjudge_ccsadmin = 8
+    # pc2 for wf2023
+    max_ccsadmin = 8
```

Comment:
Probably prefix this with pc2?

Reply:
in the 'c' script they are without pc2 too.